src/vm/docs/mod.rs

```diff
@@ -1364,6 +1368,23 @@ The `id-header-hash` is the block identifier value that must be used as input to
 "
 };
 
+const GET_BURN_BLOCK_INFO_API: SpecialAPI = SpecialAPI {
+    input_type: "BlockInfoPropertyName, uint",
+    output_type: "(optional buff)",
+    signature: "(get-burn-block-info? prop-name block-height)",
+    description: "The `get-burn-block-info?` function fetches data for a block of the given *burnchain* block height. The
+value and type returned are determined by the specified `BlockInfoPropertyName`. If the provided `block-height` does
+not correspond to an existing block prior to the current block, the function returns `none`. The only available property
```

Comment:
It will also return `none` for any burnchain block value prior to the start of the Stacks chain (i.e. 666050).  We can discuss whether or not this is desirable, but that's how the code currently behaves.